src/lib/select/select.spec.ts

```diff
@@ -2917,11 +2917,15 @@ describe('MatSelect', () => {
       const topDifference = Math.floor(optionTop) -
           Math.floor(triggerTop - triggerFontSize - triggerExtraLineSpaceAbove);
 
+
+      console.log({optionTop});
+      console.log({triggerTop}, ' - ', (triggerTop - triggerFontSize - triggerExtraLineSpaceAbove));
```

Comment:
Leftover logs?

Reply:
As mentioned on Slack. Debugging is happening right now. One specific select test is timing out on Edge Saucelabs.